sockeye/model.py

```diff
@@ -59,6 +59,8 @@ def __init__(self,
                  config_encoder: encoder.EncoderConfig,
                  config_decoder: decoder.DecoderConfig,
                  config_loss: loss.LossConfig,
+                 config_length_ratio_loss: loss.LossConfig = None,
```

Comment:
- type should be Optional[loss.LossConfig] if it can be none
- would rename to more general length_task_*

Reply:
fixed